src/components/results/sensitivity-analysis/paged-sensitivity-analysis-result.tsx

```diff
@@ -158,16 +159,17 @@ function PagedSensitivityAnalysisResult({
             offset: typeof rowsPerPage === 'number' ? page * rowsPerPage : rowsPerPage.value,
             pageSize: typeof rowsPerPage === 'number' ? rowsPerPage : rowsPerPage.value,
             pageNumber: page,
-            ...Object.fromEntries(
-                filters?.map((elem) => [
-                    DATA_KEY_TO_FILTER_KEY[elem.column as keyof typeof DATA_KEY_TO_FILTER_KEY],
-                    elem.value,
-                ])
-            ),
             ...sortSelector,
         };
+        const mappedFilters = filters?.map((elem) => {
+            const newColumn =
+                nOrNkIndex === 0
+                    ? DATA_KEY_TO_FILTER_KEY_N[elem.column as keyof typeof DATA_KEY_TO_FILTER_KEY_N]
+                    : DATA_KEY_TO_FILTER_KEY_NK[elem.column as keyof typeof DATA_KEY_TO_FILTER_KEY_NK];
```

Comment:
another way to write this : 

```suggestion
            const keyMap = nOrNkIndex === 0 ? DATA_KEY_TO_FILTER_KEY_N : DATA_KEY_TO_FILTER_KEY_NK;
            const newColumn = keyMap[elem.column as keyof typeof keyMap];
```